code/modules/recycling/disposal/outlet.dm

```diff
@@ -62,7 +62,7 @@
 	for(var/A in H)
 		var/atom/movable/AM = A
 		AM.forceMove(T)
-		AM.pipe_eject(dir)
+		SEND_SIGNAL(AM, COMSIG_MOVABLE_PIPE_EJECTING, dir)
```

Comment:
Signals must have a single source.
Yes, there are several bad examples around in the code, but we should not expand on them.